rust/storage/src/s3.rs

```diff
@@ -317,11 +319,7 @@ impl S3Storage {
         &self,
```

Comment:
nit: also remove the `#[tracing::instrument(skip(self), level = "trace")]` above?